Content/Maatregelen/M19/ICTU.md

```diff
@@ -1,3 +1,7 @@
 #### ICTU
 
 ICTU ondersteunt dit met Docker en/of virtuele machines (VM) en een VLAN per project. Een nieuwe afgeschermde digitale omgeving is binnen een werkweek na aanvraag beschikbaar.
+
+Binnen de projectomgeving bepaalt het projectteam zelf de gewenste configuratie.
+Veranderingen aan de omgeving komen alleen in overleg met de centrale beheergroep tot stand.
+De centrale beheergroep voert de veranderingen eventueel door in de referentie-omgeving, die als startpunt voor nieuwe projectomgevingen dient.
```

Comment:
De referentieomgeving komt hier uit de lucht vallen. Wat is het? Wie beheert het? Waarom is het belangrijk?